Source/WebCore/dom/Element.cpp

```diff
@@ -1560,6 +1560,15 @@ int Element::clientHeight()
     return 0;
 }
 
+double Element::currentCSSZoom()
+{
+    protectedDocument()->updateLayoutIgnorePendingStylesheets({ LayoutOptions::ContentVisibilityForceLayout }, this);
+
+    if (CheckedPtr renderer = this->renderer())
+        return renderer->style().usedZoom() / RenderStyle::initialZoom();
```

Comment:
Have you tested this works the same as Firefox when running it on a page that's zoomed away from the default value with `Cmd+` or `Cmd-`?

Reply:
I tried zoom-in and zoom-out to see difference on MDN page (play - test case) and it continues to match Mozilla Firefox.

Reply:
Can you reimplement the fixes? it has been reverted